pi_piper.gemspec

```diff
@@ -2,14 +2,14 @@
 
 Gem::Specification.new do |s|
   s.name = "pi_piper"
-  s.version = "2.0.beta.11"
+  s.version = "2.0.0"
```

Comment:
Maybe we could call it version 1.9.9 in order to deliver a quick fix, and wait 2.0.0 with some major enhancements ?
or start a 2.1 just after the 2.0.0 release. your call

enhancements could be :
- Support of PWM
- Support of two drivers bcm2835 c library & Sys class (file access)
- non root usage (at least with last version of raspian based of Lenny)